docs/github-action.md

```diff
@@ -47,9 +47,39 @@ jobs:
 
 As written, the scanner will run on 12:12 pm UTC every Monday. You can change the schedule by following the instructions [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule).
 
+### Customization
+
+`osv-scanner-reusable.yml` takes the `scan-args` parameter as input, this value is passed to `osv-scanner` CLI after being split by each line. See the [usage]()./usage) page for the available options.
```

Comment:
I think we may want to be a little more direct about what `osv-scanner-reuseable.yml` is. If I was newer to GitHub Actions I might be confused because I don't need this file in my project.

Reply:
Good point, maybe adding something like

> In your github action file (`.github/workflows/osv-scanner-scheduled.yml`)...

could point the user at the file they are meant to edit.